index.d.ts

```diff
@@ -129,6 +141,41 @@ export namespace Cmd {
   }
   export type PromiseResult<T> = T extends Promise<infer U> ? U : T;
 
+  export function run<Func extends (...args: any[]) => Promise<any> | any>(
```

Comment:
doesn't `any` include `Promise<any>` anyway? why say `any | Promise<any>`?

Reply:
Not sure - I didn't modify that part since I don't know is there's a historical reason for is.

My guess is that the author was just being more explicit to be descriptive since, yes, `any` should include `Promise<any>`.